### DISCUSSION TITLE
Show on server feature

### DISCUSSION THREAD

**Discussion by Pascal62223** [equinor/witsml-explorer#192] [Q&A] [answered]
Hi,

    What is the purpose of the "Show on server" feature which is accessible through right click on a tree node in teh left explorer ?

Pascal.

**Comment by steinsiv** [comment, marked as answer]
I believe the main purpose is to utilize the "deep-link" that when applied traverses down to the object with exact same uids as the one on hands, only on a different server.  This is useful in Data Management settings. 
I do not think all objects have this implemented though. The pre-requisite is a single selection which is naturally imposed in left tree view. (It has been enabled on single object selections in main view aswell)

**Reply by Pascal62223** [reply]
OK. Thanks. Make sense
